xbmc/addons/AddonInfo.h

```diff
@@ -101,31 +328,40 @@ namespace ADDON
     static TYPE TranslateSubContent(const std::string &content);
     //@}
 
-    std::string id;
-    TYPE type;
-    AddonVersion version{"0.0.0"};
-    AddonVersion minversion{"0.0.0"};
-    std::string name;
-    std::string license;
-    std::string summary;
-    std::string description;
-    std::string libname;
-    std::string author;
-    std::string source;
-    std::string path;
-    std::string changelog;
-    std::string icon;
-    std::map<std::string, std::string> art;
-    std::vector<std::string> screenshots;
-    std::string disclaimer;
-    ADDONDEPS dependencies;
-    std::string broken;
-    InfoMap extrainfo;
-    CDateTime installDate;
-    CDateTime lastUpdated;
-    CDateTime lastUsed;
-    std::string origin;
-    uint64_t packageSize;
+    InfoMap extrainfo; // defined here, but removed in future with cpluff
+
+  private:
+    friend class ADDON::CAddonBuilder;
```

Comment:
CAddonBuilder is the only class who currently need to set the values on CAddonInfo and only temporarily, the builder becomes on next steps complete removed. See also my other comment,